piq/msid.py

```diff
@@ -307,33 +307,34 @@ class MSID(BaseFeatureMetric):
     number of samples or different dimensionalities.
 
     Args:
-        x_features: Samples from data distribution. Shape :math:`(N_x, D_x)`
-        y_features: Samples from data distribution. Shape :math:`(N_y, D_y)`
-
-    Returns:
-        score: Scalar value of the distance between image sets features.
-
-    Reference:
+        ts: Temperature values.
+        k: Number of neighbours for graph construction.
+        m: Lanczos steps in SLQ.
+        niters: Number of starting random vectors for SLQ.
+        rademacher: True to use Rademacher distribution,
+            False - standard normal for random vectors in Hutchinson.
+        normalized_laplacian: if True, use normalized Laplacian.
+        normalize: ``'empty'`` for average heat kernel (corresponds to the empty graph normalization of NetLSD),
+            ``'complete'`` for the complete, ``'er'`` for Erdos-Renyi normalization, ``'none'`` for no normalization
+        msid_mode: ``'l2'`` to compute the L2 norm of the distance between `msid1` and `msid2`;
+            ``'max'`` to find the maximum absolute difference between two descriptors over temperature
+
+    Examples:
+        >>> loss = MSID()
+        >>> x = torch.rand(3, 3, 256, 256, requires_grad=True)
+        >>> y = torch.rand(3, 3, 256, 256)
+        >>> output = loss(x, y)
+        >>> output.backward()
+
+    References:
+        Tsitsulin, A., Munkhoeva, M., Mottin, D., Karras, P., Bronstein, A., Oseledets, I., & Müller, E. (2019).
+        The shape of data: Intrinsic distance for data distributions.
         https://arxiv.org/abs/1905.11141
     """
 
     def __init__(self, ts: torch.Tensor = torch.logspace(-1, 1, 256), k: int = 5, m: int = 10, niters: int = 100,
```

Comment:
Can we remove this `torch.logspace(-1, 1, 256)`  inside the function?
This currently leads to a very long description. 

<img width="760" alt="image" src="https://user-images.githubusercontent.com/15848838/113983011-1f9c2880-9852-11eb-8f7f-d1f5b4af3021.png">